### PR TITLE
BUFFS URINAL CAKES

### PR DESCRIPTION
**Pull request by Moltijoe**
# Why is this good for the game?
ENCOURAGES GHETTO CHEM WHICH IS INNATELY INTERESTING

:cl:  
tweak: URINAL CAKES HAVE MORE AMMONIA AND CHLORINE
/:cl:

### CODE REVIEW

**Comment by neeshacark** [comment]
i pith i nthew urinal

File: code/game/objects/structures/watercloset.dm [hunk 1]
```diff
@@ -200,7 +200,7 @@
 	icon_state = "urinalcake"
 	item_state = "urinalcake"
 	w_class = WEIGHT_CLASS_TINY
-	list_reagents = list(/datum/reagent/chlorine = 3, /datum/reagent/ammonia = 1)
+	list_reagents = list(/datum/reagent/chlorine = 10, /datum/reagent/ammonia = 5)
 	foodtype = TOXIC | GROSS
```
Comment (neeshacark) [inline code/game/objects/structures/watercloset.dm:204]: urinal cakes are yummy not gross fix this injustice

Comment (Moltijoe) [reply code/game/objects/structures/watercloset.dm:204]: I TRIED, THAT GAVE A COMPILE ERROR SO I REVERTED IT

Comment (Moltijoe) [reply code/game/objects/structures/watercloset.dm:204]: image
SEE

**Comment by hedgehog1029** [comment]
finally

**Review by SapphicOverload — APPROVED** [review]

**Pull request merged**